src/components/NavBar/NavBar.jsx

```diff
@@ -56,5 +67,20 @@ export const NavBar=()=>{
           </div>
         </div>
       </nav>
+      <section className="mobile-search-cont">
```

Comment:
I guess this is search bar in mobile devices but rather than adding extra html for the search bar for mobile you can manipulate the original with the css itself